### ISSUE TITLE
Map: recommend and demonstrate providing similar info as text

### ISSUE THREAD

**Issue by rolfsmeds** [vaadin/docs#2627]
As instance of the Map component are inherently non-screen-reader-friendly, we should recommend and demonstrate applying `aria-label`s on them and/or providing the same information in a different format.

Ideally, markers visible in the Map viewport should be listed somewhere adjacent to the map for keyboard accessibility.

---

If you think this issue is important, add a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to help the community and maintainers prioritize this issue.
